src/hooks/usePopularCities.js

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from '@tanstack/react-query';
+import fetchPopularCities from '../services/PopularCitiesServise';
+import transformCityItem from '../data/transformCityItem';
+
+const popularCitiesQueryKey = 'popularCities';
```

Comment:
let's follow constant naming convention we have chosen

```suggestion
const POPULAR_CITIES_QUERY_KEY = 'popularCities';
```

Reply:
Do we use this naming only at the top level? Can we use a camel case in function blocks and etc?